includes/template-functions.php

```diff
@@ -99,11 +99,13 @@ function sensei_start_course_form( $course_id ) {
 	if ( $prerequisite_complete ) {
 		wp_enqueue_script( 'sensei-stop-double-submission' );
 
+		$sensei_start_course_form_text = apply_filters( 'sensei_start_course_form_text', esc_html( 'Take This Course', 'sensei-lms' ) );
```

Comment:
I think we should consider calling this filter something else to more accurately describe what it filters. What about `sensei_take_course_button_text`?